src/main/kotlin/com/makarytskyi/rentcar/mongock/migration/CarMigration.kt

```diff
@@ -0,0 +1,62 @@
+package com.makarytskyi.rentcar.mongock.migration
+
+import com.makarytskyi.rentcar.model.MongoCar
+import io.mongock.api.annotations.ChangeUnit
+import io.mongock.api.annotations.Execution
+import io.mongock.api.annotations.RollbackExecution
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.Sort
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.index.Index
+
+@ChangeUnit(id = "carCollectionAndIndexes", order = "1", author = "makarytskyi.o@ajax.systems")
+class CarMigration {
+
+    @Execution
+    fun createCarCollection(mongoTemplate: MongoTemplate) {
+        if (!mongoTemplate.collectionExists(MongoCar.COLLECTION_NAME)) {
+            mongoTemplate.createCollection(MongoCar.COLLECTION_NAME)
+            log.info("Collection {} was created", MongoCar.COLLECTION_NAME)
+        }
+
+        val indexOps = mongoTemplate.indexOps(MongoCar.COLLECTION_NAME)
+        indexOps.ensureIndex(
+            Index()
+                .on("brand", Sort.Direction.ASC)
+                .on("model", Sort.Direction.ASC)
+                .named("cars_brand_model_index")
```

Comment:
It can be, but i decided to specify collection name explicitly in index. By default, it has field name and value 1 or -1, indicating ascending or descending order.

Reply:
why do you need an explicit collection name in an index? An index is created for a specific collection, so it's obsolete info

Order in index name may be helpful